setup.py

```diff
@@ -1,10 +1,32 @@
 from setuptools import find_packages, setup
 
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
 setup(
     name='fastener',
-    packages=find_packages(),
-    version='1.0.0',
-    description='desc',
+    version='1.0.3',
     author='Filip Koprivec, Klemen Kenda, Gal Petkovsek',
+    author_email='',
+    description='Feature selection enabled by entropy',
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/JozefStefanInstitute/FASTENER.git",
+    packages=["fastener"],
+    #py_modules=["fastener", "item", "random_utils"],
+    install_requires=["joblib==0.14.1",
+                    "mypy==0.761",
```

Comment:
Fix hanging indent.
Be consistent when using single or double quotes in the same file.